### PR TITLE
Remove net452 from zipkin exporter

### PR DESCRIPTION
**Pull request by cijothomas**
Part of #2138

## Changes

Please provide a brief description of the changes here.

For significant contributions please make sure you have completed the following items:

* [ ] `CHANGELOG.md` updated for non-trivial changes
* [ ] Design discussion issue #
* [ ] Changes in public API reviewed

### CODE REVIEW

**Review by CodeBlanch — APPROVED** [review]
LGTM

File: src/OpenTelemetry.Exporter.Zipkin/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Removes .NET Framework 4.5.2 support. The minimum .NET Framework version
+  supported is .NET 4.6.1. [2138](https://github.com/open-telemetry/opentelemetry-dotnet/issues/2138)
```
Comment (reyang) [inline src/OpenTelemetry.Exporter.Zipkin/CHANGELOG.md:6]:
```suggestion
  supported is .NET 4.6.1. ([#2138](https://github.com/open-telemetry/opentelemetry-dotnet/issues/2138))
```

File: src/OpenTelemetry.Exporter.Zipkin/OpenTelemetry.Exporter.Zipkin.csproj [hunk 2]
```diff
@@ -25,10 +25,4 @@
     <PackageReference Include="System.Text.Json" Version="$(SystemTextJsonPkgVer)" Condition="'$(TargetFramework)' != 'net452'" />
   </ItemGroup>
 
-  <ItemGroup Condition="'$(TargetFramework)' == 'net452'">
-    <Compile Include="$(RepoRoot)\src\OpenTelemetry\Internal\DateTimeOffsetExtensions.net452.cs" Link="Includes\DateTimeOffsetExtensions.net452.cs" />
-    <Reference Include="System.Net.Http" />
-    <PackageReference Include="Newtonsoft.Json" Version="$(NewtonsoftJsonPkgVer)" />
```
Comment (reyang) [inline src/OpenTelemetry.Exporter.Zipkin/OpenTelemetry.Exporter.Zipkin.csproj:31]: 👍

Comment (reyang) [reply src/OpenTelemetry.Exporter.Zipkin/OpenTelemetry.Exporter.Zipkin.csproj:31]: Check if we still need `NewtonsoftJsonPkgVer` to be defined or we can remove it entirely.

**Review by reyang — APPROVED** [review]

**Pull request merged**